web_src/css/modules/animations.css

```diff
@@ -102,6 +107,62 @@ code.language-math.is-loading::after {
   }
 }
 
+
+@keyframes pulse-opacity {
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .pulse {
+    animation: pulse 2s linear;
```

Comment:
You should not set the whole `animation`, but only `animation-name`. Where is this used/defined?

Reply:
The pulse animation is defined in this file, just above the `pulse-opacity` animation. It was the one used previously by the `.pulse` class.

And yeah, you're right, i'll switch to only changin the animation name!

Reply:
Ah I see, this is in use in the review box:

```
web_src/js/features/repo-diff.js:24:      $reviewBox.removeClass('pulse');
web_src/js/features/repo-diff.js:26:      $reviewBox.addClass('pulse');
```